zenpy/lib/request.py

```diff
@@ -118,6 +118,33 @@ def delete(self,
         return response
 
 
+class ArticleCRUDRequest(CRUDRequest):
+    def post(self, api_objects, *args, **kwargs):
+        self.check_type(api_objects)
+
+        create_or_update = kwargs.pop('create_or_update', False)
+        create = kwargs.pop('create', False)
+        notify_subscribers = kwargs.pop('notify_subscribers', None)
+        if isinstance(api_objects, Iterable) and create_or_update:
+            kwargs['create_or_update_many'] = True
+            endpoint = self.api.endpoint.create_or_update_many
+        elif isinstance(api_objects, Iterable):
+            kwargs['create_many'] = True
+            endpoint = self.api.endpoint
+        elif create_or_update:
+            endpoint = self.api.endpoint.create_or_update
+        elif create:
+            endpoint = self.api.endpoint.create
+        else:
+            endpoint = self.api.endpoint
+
+        payload = self.build_payload(api_objects)
+        if notify_subscribers is not None:
+            payload['notify_subscribers'] = True if notify_subscribers else False
```

Comment:
Nothing wrong with using multiple lines, if we don't fear overriding notify_subscribers (which seems to be the case from its name), this could be:

```suggestion
        payload['notify_subscribers'] = bool(notify_subscribers)
```

Reply:
I'm definitely taking this suggestion!  Nice edit.

Reply:
However, your suggestion of NOT omitting is a mistake.  If notify_subscribers is None, then the default is actually TRUE and should be omitted.  I'm preserving that logic but just taking your cast  ✨